src/device_batch_gemm.cc

```diff
@@ -92,20 +92,20 @@ void gemm(
 
             if (layout == Layout::RowMajor) {
                 // swap transA <=> transB, m <=> n, B <=> A
-                device::batch_sgemm( queue,
-                                    transB_, transA_,
-                                    n_, m_, k_,
-                                    alpha[0], dBarray, lddb_, dAarray, ldda_,
-                                    beta[0],  dCarray, lddc_,
-                                    ibatch);
+                internal::batch_gemm(
+                    transB_, transA_,
+                    n_, m_, k_,
+                    alpha[0], dBarray, lddb_, dAarray, ldda_,
```

Comment:
same as other comments: missed changing from lddb to ldb, ldda to lda and lddc to ldc

Reply:
All these for batch routines fixed in PR #22.